localization/es/singleton/README.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Singleton
+category: Creational
+language: en
```

Comment:
Change language here to `es`, like this:
```
language: es
```

Reply:
Done